src/pages/workspace/WorkspaceMoreFeaturesPage.tsx

```diff
@@ -75,7 +77,8 @@ function WorkspaceMoreFeaturesPage({policy, route}: WorkspaceMoreFeaturesPagePro
             icon: Illustrations.FolderOpen,
             titleTranslationKey: 'workspace.moreFeatures.categories.title',
             subtitleTranslationKey: 'workspace.moreFeatures.categories.subtitle',
-            isActive: policy?.areCategoriesEnabled ?? false,
+            isActive: (policy?.areCategoriesEnabled ?? false) || hasAccountingConnection,
```

Comment:
TBf the `areCategoriesEnabled` property should be set to true when the integration is set up on policy, did we cover that in a backend? ie when integration is set up, we need to also se the `areCategoriesEnabled` to true @hayata-suenaga

Reply:
[Seems like](https://github.com/Expensify/Web-Expensify/pull/40741/files) we didn't handle force enabling these features when the accounting integration is established on the backend.

If we add that logic, I agree that specifying `hasAccountingConnection` is redundant. This is a logic that front end doesn't have to be aware of.

Reply:
okay so I removed `hasAccountingConnection` from `isActive` boolean check.

Reply:
[and also created a GH issue for the backend work](https://github.com/Expensify/Expensify/issues/387834)

Reply:
Thanks